packages/flutter/lib/src/material/checkbox_list_tile.dart

```diff
@@ -388,6 +389,10 @@ class CheckboxListTile extends StatelessWidget {
   /// {@macro flutter.material.ListTile.shape}
   final ShapeBorder? shape;
 
+  /// The color of the tile if it's selected.
+  ///
```

Comment:
You can remove the extra empty doc line.

You can also define the default color for the selectedTileColor if no color is specified, or you can simply use Macro. For example, take a look at the shape property.

Reply:
I have done the changes.